test/__snapshots__/jsToAqua.test.ts.snap

```diff
@@ -0,0 +1,127 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 1`] = `
+{
+  "type": "[]u64",
+  "typeDefs": undefined,
+  "value": "[
+        1,
+        2,
+        3
+]",
+}
+`;
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 2`] = `
+{
+  "type": "[]i64",
+  "typeDefs": undefined,
+  "value": "[
+        1,
+        -2,
+        3
+]",
+}
+`;
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 3`] = `
+{
+  "type": "[]f64",
+  "typeDefs": undefined,
+  "value": "[
+        1.1,
+        2,
+        3
+]",
+}
+`;
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 4`] = `
+{
+  "type": "[][]string",
+  "typeDefs": undefined,
+  "value": "[
+        [
+            "1"
+        ],
+        [
+            "2"
+        ],
+        [
+            "3"
+        ]
+]",
+}
+`;
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 5`] = `
+{
+  "type": "Res",
+  "typeDefs": "data Res:
+    a: i64
+    b: f64",
+  "value": "Res(
+        a=-1,
+        b=2.2
+    )",
+}
+`;
+
+exports[`Conversion from js to aqua Expect objects to match snapshot 6`] = `
+{
+  "type": "Res",
+  "typeDefs": "data ResA:
+    b: []u64
+
+data ResE:
+    f: string
+
+data Res:
+    a: ResA
+    c: ?u8
+    d: ?u8
+    e: []ResE",
+  "value": "Res(
+        a=ResA(
+            b=[
+                2
+            ]
+        ),
+        c=nil,
+        d=nil,
```

Comment:
Let's make at least on of optionals non-nil.

Reply:
I reshuffeled test cases in such a way that it's now visible were there are nulls and when they happen and where are the optional values and when they happen including the case when optional is not null